### ISSUE TITLE
Autarky and Disiregime not having election options

### ISSUE THREAD

**Issue by LordNotix** [vic2mod/ParadoxPlazaMod#66]
You can't encourage your population to care about these options

**Comment by willwill54** [comment]
Yeah this needs to be added but it might need to be done so the trade policy and economic policy election events change after fascism is invented since it makes no sense for it to exist before. But I was also thinking maybe we could go back to vanilla election events since they look better and give more information or have an option the game to play with one or the other

**Comment by willwill54** [comment]
Yeah this needs to be added but it might need to be done so the trade policy and economic policy election events change after fascism is invented since it makes no sense for it to exist before. But I was also thinking maybe we could go back to vanilla election events since they look better and give more information or have an option the game to play with one or the other
